native_client/dotnet/DeepSpeechConsole/DeepSpeechConsole.csproj

```diff
@@ -35,8 +35,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="NAudio, Version=1.8.5.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\NAudio.1.8.5\lib\net35\NAudio.dll</HintPath>
+    <Reference Include="NAudio, Version=2.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Using `2.0.0` forces a failure at finding NAudio, so:
 - 2.0.0.0 was proper
 - NAudio 2.0.0.0 was correctly found
 - We were still having missing reference for WaveBuffer / WaveFileReader yet we had `using NAudio.Wave`